devices/src/lib.rs

```diff
@@ -41,6 +41,14 @@ pub type DeviceEventT = u16;
 pub enum EpollHandlerPayload {
     /// DrivePayload(disk_image)
     DrivePayload(File),
+    /// Used to mutate current RateLimiter settings. The buckets are rx_bytes, rx_ops,
+    /// tx_bytes, and tx_ops, respectively.
+    PatchRateLimiters(
```

Comment:
Either name this `NetRateLimiterPayload` or something clearly identifying it's a _network device_ specific event payload, or create a device-generic `RateLimiterPayload` that has a `vec!` or other variable sized struct of rate limiters so the event can be used for both **net** and **drive**.

In the generic case, the handler will validate the payload and complain if the contents are wrong.

Reply:
Renamed to `NetRateLimiterPayload`.